docs/03_server/10_integration/12_open-api.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Open API'
+sidebar_label: 'Open API'
+id: open-api
+keywords: [server, integration, open api, introduction]
+tags:
+  - server
+  - integration
+  - open api
+  - introduction
+---
+
+:::important
+
+Open API support was introduced in version 7.0 of the Genesis platform.
+
+:::
+
+The Genesis low-code platform supports the use of [Open API](https://swagger.io/specification/) in order to integrate with external systems. 
+
+Genesis makes this easy to configure and set up, and an Open API specification is generated for the following resources:
+
+* [Request Server](../../../../server/request-server/introduction/)
+  * The specification is generated for every GPAL request server.
+* [Event Handler](../../../../server/event-handler/introduction/)
```

Comment:
maybe we could show a sample event handler definition and the result in swagger or openapi spec.